### ISSUE TITLE
Change search spinner message

### ISSUE THREAD

**Issue by zmitchell** [flox/flox#1101]
The current spinner shown while the search index is building looks like this:

```text
Searching for packages...    <blank space> This might take a while the first time you run it
```

The "this might take a while" part of the message always appears on the right side of the terminal, which looks funny to me and gives the impression that we've messed up the styling somehow.

The "the first time" part of the message just isn't correct, you'll see that spinner every time you run a `flox update`.

By saying "searching for packages" while we show the spinner it gives the impression that the search process is really slow, but that's not the case, it's building the search index that's slow. A developer likely knows what that "building a search index" means and will understand that it could be slow, whereas a slow "search" is likely to give the impression that the software is malfunctioning.

**Acceptance criteria:**
- Remove the blank space between the two parts of the spinner message
- Change the message to read `Building the search index, this may take up to a minute`
    - The "up to a minute" part is there because "a while" gives no indication to the user whether the operation could take 10 seconds or 10 minutes, so they have no indication as to whether the software is working as intended.

**Comment by mkenigs** [comment]
> The "the first time" part of the message just isn't correct, you'll see that spinner every time you run a `flox update`.

From discussion it seems the message appeared for @zmitchell after a `flox update` because he manually removed global-manifest.lock. So I don't think we should use that as a justification for changing the message

**Comment by ghudgins** [comment]
FWIW besides @zmitchell's personal experience I have not witnessed any issues with users understanding the right-aligned message (and no one thought it was broken). I also haven't ever observed the wrong message on `flox update` (just another ++ to Matthew's point). Also `flox update` is planned for deprecation because of the upcoming catalog service so that point is a bit moot. 

However the proposed solution is fine and I do agree the language isn't as precise as it could be...the proposal has more specific language which is a good thing.

tl;dr - lets proceed

**Comment by zmitchell** [comment]
This ticket is outdated, we don't build search indexes locally anymore.
